android/2023-emmsale/app/src/main/java/com/emmsale/presentation/KerdyApplication.kt

```diff
@@ -28,6 +32,63 @@ class KerdyApplication : Application() {
         firebaseAnalytics = FirebaseAnalytics.getInstance(this)
     }
 
+    private fun initNotificationChannels() {
+        val notificationManager = getSystemService(NOTIFICATION_SERVICE) as NotificationManager
+
+        notificationManager.createNotificationChannels(
+            listOf(
+                createFollowNotificationChannel(),
+                createChildCommentNotificationChannel(),
+                createInterestEventNotificationChannel(),
+            ),
+        )
```

Comment:
용도에 따라 사용자가 수신 거부를 설정할 수 있도록 채널을 잘 만들어주셨네요!
listOf()를 사용하는 것도 좋지만, depth가 1 깊어진다는 점이 조금 걸립니다.

vararg를 사용한 코드는 어떠신가요?

```kotlin
 private fun initNotificationChannels() {
    val notificationManager = getSystemService(NOTIFICATION_SERVICE) as NotificationManager

    notificationManager.createNotificationChannels(
        createFollowNotificationChannel(),
        createChildCommentNotificationChannel(),
        createInterestEventNotificationChannel(),
    )
}

private fun NotificationManager.createNotificationChannels(vararg channels: NotificationChannel) {
    channels.forEach { createNotificationChannel(it) }
}
```

Reply:
depth가 깊어져서 가독성이 떨어지는 점을 캐치해주신 점에 대해 감사합니다.
```suggestion
        val notificationChannels = listOf(
            createFollowNotificationChannel(),
            createChildCommentNotificationChannel(),
            createInterestEventNotificationChannel(),
        )
        notificationManager.createNotificationChannels(notificationChannels)
```
저는 개인적으로 함수의 개수가 많아진다면 클래스 내에서 원하는 함수를 찾을 때 힘들게 만들 수 있다고 생각합니다. 그래서 함수가 로직을 추상화하지 않는다면 불필요하게 함수를 만들지 않으려고 노력합니다. 지역 변수로 선언 후 depth를 얕게 만드는 것은 어떤가요?